Dockerfile.centos

```diff
@@ -0,0 +1,56 @@
+FROM centos:7 AS build-env
+LABEL maintainer "Sida Chen <sidchen@redhat.com>"
+
+ARG GIT_TAG
+RUN test -n "$GIT_TAG" # Expect Clair GIT TAG to be non-empty.
+
+# Install Golang by retrieving the binary
+ENV GO_VERSION=1.11.5
+ENV GO_OS=linux
+ENV GO_ARC=amd64
+RUN curl https://dl.google.com/go/go${GO_VERSION}.${GO_OS}-${GO_ARC}.tar.gz --output go.tar.gz
+RUN echo ff54aafedff961eb94792487e827515da683d61a5f9482f668008832631e5d25 go.tar.gz > GOCHECKSUM
+RUN sha256sum -c GOCHECKSUM
+RUN tar -C /usr/local -xzf go.tar.gz > /dev/null
+ENV GOPATH=/go
+ENV PATH=$PATH:/usr/local/go/bin:${GOPATH}/bin
+
+# Verify go verion
+RUN go version
+
+# Install dependencies
+RUN yum install -y git bzr rpm xz gcc
+# musl-dev 
+RUN go get -u github.com/cloudflare/cfssl/cmd/cfssl
+RUN go get -u github.com/cloudflare/cfssl/cmd/cfssljson
```

Comment:
We should probably check out specific stable releases of this.

I think `go get` will probably do it for us if we were using go modules.